r/man/read_delim_arrow.Rd

```diff
@@ -109,7 +109,7 @@ parsing options provided in other arguments (e.g. \code{delim}, \code{quote}, et
 \link[=Table]{arrow::Table}?}
 }
 \value{
-A \code{data.frame}, or an Table if \code{as_data_frame = FALSE}.
+A \code{data.frame}, or a Table if \code{as_data_frame = FALSE}.
```

Comment:
This is a generated file, so you should fix the source too: https://github.com/apache/arrow/blob/master/r/R/csv.R#L70

Reply:
Thank you for your comment. I addressed this.